examples/algorithms/randomforest/randomforest.py

```diff
@@ -0,0 +1,130 @@
+from numpy.random import rand
```

Comment:
Would  you mind adding the Boost license information (and your copyright) to this file?

Reply:
sure thing!